README.md

```diff
@@ -220,6 +220,6 @@ I would like to thank Microverse and my Coding Partners.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It appears that the MIT link is still broken and leads to a "404 page not found" error and this is because you don't have a file named `MIT.md` at the root of your repository so I'd highly advise you fix this by adding the proper file.